tests/_support/rest-api-tester/assets/js/src/trap-script.js

```diff
@@ -0,0 +1,145 @@
+(function ($, undefined) {
```

Comment:
Our JS standards use the same spacing as our PHP standards. Add spaces around parens as per those standards :)